canonical-web-and-design/maas-ui
# Build CPUPopover component.

## Done

- Built CPUPopover component according to [design](https://app.zeplin.io/project/5ed922c192bf80b9c37cb4fb/screen/5efa229d3f80d013734e9b21)
- Originally I tried to build this as a generic Popover component that could be used elsewhere, but I think this particular design is too specific to cores.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to /MAAS/r/kvm and hover over one of the CPU core meters.
- Check that the CPUPopover component shows up with the correct data.

## Fixes

Fixes canonical-web-and-design/MAAS-squad#1952

## Screenshot
![2020-07-07_17-42](https://user-images.githubusercontent.com/25733845/86740750-38d13800-c07a-11ea-8900-2a01656e989a.png)
